### ISSUE TITLE
git trac SUBCOMMAND -h

### ISSUE THREAD

**Issue by gagern** [sagemath/git-trac-command#21]
There is some inconsistency in `-h` vs. `--help`:

    $ git trac --help
    No manual entry for git-trac

    $ git trac -h
    ....

    $ git trac pull -h
    usage: git-trac pull [-h] [ticket_or_branch]
    git-trac pull: error: argument ticket_or_branch: invalid TicketOrBranch value: 'help'
    2
    Exiting.

    $ git trac pull --help
    usage: git-trac pull [-h] [ticket_or_branch]

    positional arguments:
      ticket_or_branch  Ticket number or remote branch name

    optional arguments:
      -h, --help        show this help message and exit

It would be great if all subcommands were to support the `-h` switch, as the online help claims they should.

It would be even greater if we had manual pages for `--help` written and made available on the path through the add-on, but I don't know if the latter is even possible.

**Comment by vbraun** [comment]
Manpages need to be installed in a different path than $PATH, so it would be difficult to install.

**Comment by novoselt** [comment]
But supporting -h would be nice, I just got the following:
```bash
novoselt@sage:~/sage$ git trac create
usage: git-trac create [-h] [-b BRANCH_NAME] summary
git-trac create: error: too few arguments
2
Exiting.
novoselt@sage:~/sage$ git trac create -h
Remote branch: u/novoselt/help
Newly-created ticket number: 17834
Ticket URL: http://trac.sagemath.org/17834
Local branch: t/17834/help
```

**Comment by vbraun** [comment]
Fixed in 0af20cd084089b6548405d133290675fde6d7790
